2015-07-17 18:42:16 Error: [PDOException] SQLSTATE[HY000] [1045] Access denied for user 'my_app'@'localhost' (using password: YES)
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Driver\PDODriverTrait.php(48): PDO->__construct('mysql:host=loca...', 'my_app', 'secret', Array)
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(89): Cake\Database\Driver\Mysql->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(46): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(62): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(44): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Connection.php(319): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(819): Cake\Database\Connection->schemaCollection()
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(795): Bake\Shell\Task\ModelTask->_getAllTables()
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(187): Bake\Shell\Task\ModelTask->listAll()
#10 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(110): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#11 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(96): Bake\Shell\Task\ModelTask->bake('UserTypes')
#12 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\BakeShell.php(254): Bake\Shell\Task\ModelTask->main('UserTypes')
#13 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Collection\CollectionTrait.php(51): Bake\Shell\BakeShell->Bake\Shell\{closure}('user_types', 0)
#14 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\BakeShell.php(257): Cake\Collection\Collection->each(Object(Closure))
#15 [internal function]: Bake\Shell\BakeShell->all('user_types')
#16 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\Shell.php(380): call_user_func_array(Array, Array)
#17 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(204): Cake\Console\Shell->runCommand(Array, true)
#18 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(175): Cake\Console\ShellDispatcher->_dispatch()
#19 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(126): Cake\Console\ShellDispatcher->dispatch()
#20 C:\xampp\htdocs\PROJETOS\Shopping\bin\cake.php(33): Cake\Console\ShellDispatcher::run(Array)
#21 {main}


2015-07-28 12:48:56 Error: [Cake\Console\Exception\MissingShellException] Shell class for "Serve" could not be found.
Exception Attributes: array (
  'class' => 'Serve',
)
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(201): Cake\Console\ShellDispatcher->findShell('serve')
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(175): Cake\Console\ShellDispatcher->_dispatch()
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(126): Cake\Console\ShellDispatcher->dispatch()
#3 C:\xampp\htdocs\PROJETOS\Shopping\bin\cake.php(33): Cake\Console\ShellDispatcher::run(Array)
#4 {main}


2015-07-28 18:00:43 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'shopping.subscribers' doesn't exist
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'subscribers', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(65): Cake\Database\Schema\Collection->describe('subscribers', Array)
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\ORM\Table.php(426): Cake\Database\Schema\CachedCollection->describe('subscribers')
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\ORM\Table.php(502): Cake\ORM\Table->schema()
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(195): Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(110): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(96): Bake\Shell\Task\ModelTask->bake('Subscribers')
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\BakeShell.php(254): Bake\Shell\Task\ModelTask->main('Subscribers')
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Collection\CollectionTrait.php(51): Bake\Shell\BakeShell->Bake\Shell\{closure}('subscribers', 0)
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\bake\src\Shell\BakeShell.php(257): Cake\Collection\Collection->each(Object(Closure))
#10 [internal function]: Bake\Shell\BakeShell->all('subscribers')
#11 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\Shell.php(380): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(204): Cake\Console\Shell->runCommand(Array, true)
#13 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(175): Cake\Console\ShellDispatcher->_dispatch()
#14 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(126): Cake\Console\ShellDispatcher->dispatch()
#15 C:\xampp\htdocs\PROJETOS\Shopping\bin\cake.php(33): Cake\Console\ShellDispatcher::run(Array)
#16 {main}


2015-09-02 13:40:58 Error: [Cake\Core\Exception\Exception] Unable to insert fixtures for "App\Test\TestCase\Model\Table\UsersTableTest" test case. SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it.

Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(144): Cake\TestSuite\Fixture\FixtureManager->load(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#9 {main}


2015-09-02 17:40:18 Warning: Warning (4096): Argument 2 passed to Cake\ORM\Table::patchEntity() must be of the type array, integer given, called in C:\xampp\htdocs\PROJETOS\Shopping\src\Controller\UsersController.php on line 81 and defined in [C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\ORM\Table.php, line 2026]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2026
App\Controller\UsersController::add() - APP/Controller\UsersController.php, line 81
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 411
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
Cake\TestSuite\IntegrationTestCase::_sendRequest() - CORE\src\TestSuite\IntegrationTestCase.php, line 273
Cake\TestSuite\IntegrationTestCase::post() - CORE\src\TestSuite\IntegrationTestCase.php, line 201
App\Test\TestCase\Controller\UsersControllerTest::testAdd() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 61
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-02 17:40:18 Warning: Warning (4096): Argument 2 passed to Cake\ORM\Marshaller::merge() must be of the type array, integer given, called in C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\ORM\Table.php on line 2032 and defined in [C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 419]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
Cake\ORM\Marshaller::merge() - CORE\src\ORM\Marshaller.php, line 419
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2032
App\Controller\UsersController::add() - APP/Controller\UsersController.php, line 81
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 411
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
Cake\TestSuite\IntegrationTestCase::_sendRequest() - CORE\src\TestSuite\IntegrationTestCase.php, line 273
Cake\TestSuite\IntegrationTestCase::post() - CORE\src\TestSuite\IntegrationTestCase.php, line 201
App\Test\TestCase\Controller\UsersControllerTest::testAdd() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 61
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 12:19:48 Error: [Cake\Core\Exception\Exception] Unable to insert fixtures for "App\Test\TestCase\Controller\UsersControllerTest" test case. SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it.

Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(144): Cake\TestSuite\Fixture\FixtureManager->load(Object(App\Test\TestCase\Controller\UsersControllerTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Controller\UsersControllerTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Controller\UsersControllerTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Controller\UsersControllerTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#9 {main}


2015-09-03 14:34:30 Warning: Warning (2): file_get_contents(/users/edit/900000): failed to open stream: No such file or directory in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 162]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 162
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:37:12 Warning: Warning (2): file_get_contents(/users/edit/900000): failed to open stream: No such file or directory in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 162]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 162
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:38:10 Warning: Warning (2): file_get_contents(http://localhost:8765/users/edit/900000): failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found
 in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 156]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 156
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:38:41 Warning: Warning (2): file_get_contents(http://localhost:8765/users/edit/900002): failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found
 in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 156]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 156
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:48:14 Warning: Warning (2): file_get_contents(/users/edit/1): failed to open stream: No such file or directory in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 163]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 163
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:49:26 Warning: Warning (2): file_get_contents(/users/edit/1): failed to open stream: No such file or directory in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 162]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 162
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:54:50 Warning: Warning (2): file_get_contents(/1/users/edit/900000): failed to open stream: No such file or directory in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 165]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 165
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 14:55:18 Warning: Warning (2): file_get_contents(/1/users/edit/1): failed to open stream: No such file or directory in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 165]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 165
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-03 17:08:08 Warning: Warning (2): file_get_contents(http://localhost:8765/users/edit/900000): failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found
 in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 158]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 158
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-04 13:58:46 Error: [Cake\Core\Exception\Exception] Unable to insert fixtures for "App\Test\TestCase\Model\Table\UsersTableTest" test case. SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it.

Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(144): Cake\TestSuite\Fixture\FixtureManager->load(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Model\Table\UsersTableTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#9 {main}


2015-09-04 14:23:36 Error: [UnexpectedValueException] Referenced fixture class "App\Test\Fixture\ProductMediasFixture" not found. Fixture "app.product_medias" was referenced in test case "App\Test\TestCase\Controller\BannersControllerTest".
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureManager.php(67): Cake\TestSuite\Fixture\FixtureManager->_loadFixtures(Object(App\Test\TestCase\Controller\BannersControllerTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(143): Cake\TestSuite\Fixture\FixtureManager->fixturize(Object(App\Test\TestCase\Controller\BannersControllerTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Controller\BannersControllerTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Controller\BannersControllerTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Controller\BannersControllerTest))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#10 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#11 {main}


2015-09-04 14:25:30 Error: [UnexpectedValueException] Referenced fixture class "App\Test\Fixture\ProductMediasFixture" not found. Fixture "app.product_medias" was referenced in test case "App\Test\TestCase\Controller\BannersControllerTest".
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureManager.php(67): Cake\TestSuite\Fixture\FixtureManager->_loadFixtures(Object(App\Test\TestCase\Controller\BannersControllerTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(143): Cake\TestSuite\Fixture\FixtureManager->fixturize(Object(App\Test\TestCase\Controller\BannersControllerTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Controller\BannersControllerTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Controller\BannersControllerTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Controller\BannersControllerTest))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#10 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#11 {main}


2015-09-04 14:26:37 Error: [UnexpectedValueException] Referenced fixture class "App\Test\Fixture\ProductMediasFixture" not found. Fixture "app.product_medias" was referenced in test case "App\Test\TestCase\Controller\BookingsControllerTest".
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureManager.php(67): Cake\TestSuite\Fixture\FixtureManager->_loadFixtures(Object(App\Test\TestCase\Controller\BookingsControllerTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(143): Cake\TestSuite\Fixture\FixtureManager->fixturize(Object(App\Test\TestCase\Controller\BookingsControllerTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Controller\BookingsControllerTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Controller\BookingsControllerTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Controller\BookingsControllerTest))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#10 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#11 {main}


2015-09-04 14:30:02 Error: [Cake\Core\Exception\Exception] Unable to insert fixtures for "App\Test\TestCase\Controller\PromotionProductsControllerTest" test case. SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'promotion_price' cannot be null
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(144): Cake\TestSuite\Fixture\FixtureManager->load(Object(App\Test\TestCase\Controller\PromotionProductsControllerTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Controller\PromotionProductsControllerTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Controller\PromotionProductsControllerTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Controller\PromotionProductsControllerTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#10 {main}


2015-09-04 14:33:58 Warning: Warning (2): file_get_contents(http://localhost:8765/users/edit/1): failed to open stream: No connection could be made because the target machine actively refused it.
 in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 147]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 147
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



2015-09-04 14:34:00 Error: [UnexpectedValueException] Referenced fixture class "App\Test\Fixture\ProductMediasFixture" not found. Fixture "app.product_medias" was referenced in test case "App\Test\TestCase\Model\Table\BannersTableTest".
Stack Trace:
#0 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureManager.php(67): Cake\TestSuite\Fixture\FixtureManager->_loadFixtures(Object(App\Test\TestCase\Model\Table\BannersTableTest))
#1 C:\xampp\htdocs\PROJETOS\Shopping\vendor\cakephp\cakephp\src\TestSuite\Fixture\FixtureInjector.php(143): Cake\TestSuite\Fixture\FixtureManager->fixturize(Object(App\Test\TestCase\Model\Table\BannersTableTest))
#2 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(325): Cake\TestSuite\Fixture\FixtureInjector->startTest(Object(App\Test\TestCase\Model\Table\BannersTableTest))
#3 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestResult.php(541): PHPUnit_Framework_TestResult->startTest(Object(App\Test\TestCase\Model\Table\BannersTableTest))
#4 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestCase.php(702): PHPUnit_Framework_TestResult->run(Object(App\Test\TestCase\Model\Table\BannersTableTest))
#5 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestCase->run(Object(PHPUnit_Framework_TestResult))
#6 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\Framework\TestSuite.php(735): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#7 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(432): PHPUnit_Framework_TestSuite->run(Object(PHPUnit_Framework_TestResult))
#8 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(148): PHPUnit_TextUI_TestRunner->doRun(Object(PHPUnit_Framework_TestSuite), Array)
#9 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\src\TextUI\Command.php(100): PHPUnit_TextUI_Command->run(Array, true)
#10 C:\xampp\htdocs\PROJETOS\Shopping\vendor\phpunit\phpunit\phpunit(47): PHPUnit_TextUI_Command::main()
#11 {main}


2015-09-04 14:40:25 Warning: Warning (2): file_get_contents(http://localhost:8765/users/edit/1): failed to open stream: No connection could be made because the target machine actively refused it.
 in [C:\xampp\htdocs\PROJETOS\Shopping\tests\TestCase\Controller\UsersControllerTest.php, line 147]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
file_get_contents - [internal], line ??
App\Test\TestCase\Controller\UsersControllerTest::testEdit() - ROOT\tests\TestCase\Controller\UsersControllerTest.php, line 147
ReflectionMethod::invokeArgs() - [internal], line ??
PHPUnit_Framework_TestCase::runTest() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 881
PHPUnit_Framework_TestCase::runBare() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 746
PHPUnit_Framework_TestResult::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestResult.php, line 601
PHPUnit_Framework_TestCase::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestCase.php, line 702
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_Framework_TestSuite::run() - ROOT\vendor\phpunit\phpunit\src\Framework\TestSuite.php, line 735
PHPUnit_TextUI_TestRunner::doRun() - ROOT\vendor\phpunit\phpunit\src\TextUI\TestRunner.php, line 432
PHPUnit_TextUI_Command::run() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 148
PHPUnit_TextUI_Command::main() - ROOT\vendor\phpunit\phpunit\src\TextUI\Command.php, line 100
[main] - ROOT\vendor\phpunit\phpunit\phpunit, line 47



